src/Apps/Legion/Services/WizardTypes.utils.ts

```diff
@@ -11,19 +11,35 @@ import {
     IViewType
 } from '../Models';
 
+// #region Base
+
+function initializeBase(args: {
```

Comment:
```suggestion
function initializeBase(args: IBase): IBase
```

Reply:
I didn't export the base class since I didn't want anything to take a dependency on that low of an interface.

Reply:
okay